O/OpenLSTO/bundled/makefile_FEA

```diff
@@ -0,0 +1,49 @@
+
+CPPFLAGS = -std=c++14 -Wall -fopenmp -w -O3 -fPIC -fopenmp
```

Comment:
Note that CPPFLAGS are typically flags for the preprocessor. Here you likely want to use CXXFLAGS

Reply:
Oops, thought I had fixed that.

Reply:
Also, `-Wall` and `-w` are kind of opposite?